src/coreclr/inc/sstring.inl

```diff
@@ -636,7 +636,7 @@ inline const UTF8 *SString::GetUTF8() const
 }
 
 // Normalize the string to unicode.  This will make many operations nonfailing.
-inline void SString::Normalize() const
+inline void SString::Normalize()
```

Comment:
I'm assuming you are leaving `ConvertToUnicode()` and `SetNormalized()` marked `const` for now to avoid one giant 'touch everything' change?

Reply:
Yes, that was something I tried, but it made me sad.